### PR TITLE
 Add AUTHORIZED_FETCH disclaimer to Actor Profile error messages

### PR DESCRIPTION
**Pull request by akhileshthite**
Screenshot 2024-05-15 at 12 00 32 AM

### CODE REVIEW

File: outbox.js [hunk 1]
```diff
@@ -170,6 +170,7 @@ class DistributedActivity extends HTMLElement {
   displayUnimplemented () {
     const message = `Activity type ${this.activityType} is not implemented yet.`
     const messageElement = document.createElement('p')
+    messageElement.classList.add('other-activity')
```
Comment (akhileshthite) [inline outbox.js:173]: Screenshot 2024-05-14 at 11 43 58 PM

File: profile.html [hunk 2]
```diff
@@ -22,18 +22,8 @@
...
-    const nextButton = document.getElementById("nextPage");
-
-    if (prevButton) {
-      prevButton.addEventListener("click", () => outboxElement.prevPage());
-    }
-    if (nextButton) {
-      nextButton.addEventListener("click", () => outboxElement.nextPage());
-    }
+  document.addEventListener('usernameUpdated', (event) => {
+    document.title = `@${event.detail.username}`;
```
Comment (akhileshthite) [inline profile.html]: Screenshot 2024-05-16 at 11 55 33 PM

File: theme-selector.js [hunk 3]
```diff
@@ -51,28 +51,37 @@ class ThemeSelector extends HTMLElement {
...
+    ['light', 'dark'].forEach(text => {
+      const option = document.createElement('option')
+      option.value = text
+      option.textContent = text.charAt(0).toUpperCase() + text.slice(1)
+      standardGroup.appendChild(option)
+    })
+    select.appendChild(standardGroup)
+
+    // Create and append the colorblind theme options within an optgroup
+    const colorBlindGroup = document.createElement('optgroup')
```
Comment (akhileshthite) [inline theme-selector.js:66]: Screenshot 2024-05-16 at 11 53 17 PM

File: actor-profile.js [hunk 4]
```diff
@@ -64,11 +83,13 @@ class ActorProfile extends HTMLElement {
     }
 
     if (actorInfo.preferredUsername) {
-      const pUserName = document.createElement('a')
+      const pUserName = document.createElement('div')
       pUserName.classList.add('profile-username')
-      pUserName.href = db.getObjectPage(actorInfo)
```
Comment (RangerMauve) [inline actor-profile.js:69]: Why was this removed? Could we add it back? I find it useful to be able to navigate to accounts on their native web url.

File: profile.html (hunk 2 above)
Comment (RangerMauve) [inline profile.html]: Instead of replacing can we do a `+=` so we can still identify the tab as being part of the social reader?

**Review by RangerMauve — CHANGES_REQUESTED** [review]
Looks good other than the one comment.

File: actor-profile.js (hunk 4 above)
Comment (akhileshthite) [reply actor-profile.js:69]: Added back.
It was a mistake.

File: actor-profile.js [hunk 5]
```diff
@@ -85,6 +85,7 @@ class ActorProfile extends HTMLElement {
     if (actorInfo.preferredUsername) {
       const pUserName = document.createElement('div')
```
Comment (RangerMauve) [inline actor-profile.js]:
```suggestion
      const pUserName = document.createElement('a')
```

This needs to be an anchor if we want href to work.

**Review by RangerMauve — CHANGES_REQUESTED** [review]

**Review by RangerMauve — APPROVED** [review]
Good job! TY
